src/ducks/apps/components/Discover.jsx

```diff
@@ -32,12 +32,8 @@ export class Discover extends Component {
           </h3>
           <div className='sto-discover-get-started-list'>
             {!isFetching && apps.map(app => {
-              // don't display app installed but not in registry
-              if (!app.isInRegistry) return null
-              let version = app.version
-              if (app.versions && Array.isArray(app.versions.stable)) {
-                version = (app.versions.stable && app.versions.stable[app.versions.stable.length - 1])
-              }
+              const stableVers = app.versions.stable
+              const version = stableVers[stableVers.length - 1]
```

Comment:
```js
const version = stableVers.pop()
```

Reply:
Since `stableVers` is just a reference to `app.versions.stable`, I am afraid that will mute the `app.versions.stable` array here and that could be error prone :/